api/src/main/java/com/okta/idx/sdk/api/client/IDXAuthenticationWrapper.java

```diff
@@ -560,6 +561,16 @@ public IDXClientContext getClientContext() throws ProcessingException {
         }
     }
 
+    /** Cancel the transaction with supplied state handle **/
+    public void cancel(String stateHandle) {
```

Comment:
I didn't see any new tests for this. Can we add one?

Reply:
This API is out of scope in this PR, will take it out.